docs/user-guide/set-up-from-template.rst

```diff
@@ -4,58 +4,47 @@
 Creating an app from the template
 #################################
 
-The best way to create a new Safir-based application for Roundtable_ is with the fastapi_safir_app_ template.
-The quickest way to do this is with the ``@sqrbot-jr create project`` Slack command.
-This page describes how to manually use the template to set up a new application.
+The best way to create a new Safir-based application is with the fastapi_safir_app_ template.
+The quickest way to do this is by running the ``/msg squarebot create project`` Slack command on the Rubin project Slack.
 
-The question for either creation method about deployment with Helm may require some further clarification.
-By default, the generated app will include Kustomize deployment configuration in a ``manifests`` directory.
-However, if you are planning on writing a Helm chart for it instead (normally in the `charts repository <https://github.com/lsst-sqre/charts>`__), this configuration is not useful.
-If you indicate the app will be deployed with Helm, the ``manifests`` directory will be omitted so that it won't clutter your app with unused files.
+For a manual equivalent, see :doc:`template-manual`.
 
-1. Install templatekit
-======================
+1. Install necessary prerequisites
+==================================
 
-In a clean Python virtual environment, install templatekit_:
+For local development, you will need Python 3.13 or later installed locally.
+The fastapi_safir_app_ template does not support earlier versions.
 
-.. prompt:: bash
-
-   python -m pip install templatekit
+You will also need to have a recent version of uv_ installed locally.
+See the `uv documentation <https://docs.astral.sh/uv/getting-started/installation/>`__ for installation instructions.
 
-2. Start the project with the template
-======================================
+2. Create the project
+=====================
 
-Next, you'll actually create the project files using the template.
-
-.. prompt:: bash
+To create a new project, run the command ``/msg squarebot create project`` on the Rubin Observatory internal project Slack.
```

Comment:
I'd be tempted to put this in a code-block (if not a prompt but i guess it'd not a traditional command line prompt) just because people will scan for it. Also keep in mind the username formatting:

```rst
.. code-block:: text
   :caption: rubin-obs.slack.com

   /msg @Squarebot create project
```